src/App/translations/en.ts

```diff
@@ -1,6 +1,10 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Can we disable a specific rule instead of everything?

Reply:
This is only disabling prettier, which is the automatic code formatter (it disliked the long line).

Other eslint rules are still on.